test/containers/NimbledroidTableSummary.test.jsx

```diff
@@ -6,20 +6,18 @@ const nimbledroidData = require('../mocks/nimbledroidData.json');
 
 it('renders correctly', () => {
   const tree = renderer
-    .create((
+    .create(
       <NimbledroidSummaryTable
         nimbledroidData={nimbledroidData}
         configuration={{
           baseProduct: 'org.mozilla.klar',
           compareProduct: 'com.chrome.beta',
-          products: [
-            'org.mozilla.klar',
-            'com.chrome.beta',
-          ],
+          products: ['org.mozilla.klar', 'com.chrome.beta'],
```

Comment:
@aimenbatool  I am not sure if I like how multi-line collapsed to single line.

"resolve" this comment when you have read it.  no action required.